components/shared_code/src/shared/model/metric.py

```diff
@@ -91,7 +91,11 @@ def issue_ids(self) -> list[str]:
 
     def addition(self) -> Callable:
         """Return the addition operator of the metric: sum, min, or max."""
-        addition = self.get("addition") or self.__data_model["metrics"][self.type()]["addition"]
+        addition = self.__data_model["metrics"][self.type()].get("addition", "sum")
+        if addition in ("min", "max") and (actual_direction := self.get("direction")):
```

Comment:
I get it, but it took a few re-reads. Perhaps something to explain in the docstring?